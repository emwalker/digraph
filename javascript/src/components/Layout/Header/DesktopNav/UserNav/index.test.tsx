import React from 'react'
import { shallow } from 'enzyme'

import { UnwrappedUserNav as UserNav } from './index'

const props = {
  viewer: {},
}

describe('<UserNav />', () => {
  // @ts-expect-error
  const wrapper = shallow(<UserNav {...props} />)

  it('renders', () => {
    expect(wrapper).toMatchSnapshot()
  })

  it('includes a "Review" link', () => {
    const link = wrapper.find('Link')
    expect(link).toBeTruthy()
    expect(link.prop('to')).toEqual('/review')
  })
})
