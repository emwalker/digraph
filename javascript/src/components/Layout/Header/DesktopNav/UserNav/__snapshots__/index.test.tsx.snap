// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<UserNav /> renders 1`] = `
<Fragment>
  <Link
    className="Link--primary px-2"
    to="/review"
  >
    Review
  </Link>
  <Relay(UserDropdown)
    viewer={Object {}}
  />
</Fragment>
`;
